examples/helloworld/src/main/java/example/HelloWorld.java

```diff
@@ -28,9 +30,11 @@
 public class HelloWorld {
 
   public static void main(String[] args) throws URISyntaxException, IOException {
```

Comment:
The previous code did not work inside a JAR, as it was complaining that there is no "file system" inside it. The new code works both inside and outside a JAR.